src/App.php

```diff
@@ -14,6 +14,10 @@
 
 use Exception;
 
+define ('APP_MODE_NORMAL', 0);
+define ('APP_MODE_INSTALL', 1);
+define ('APP_MODE_MAINTENANCE', 2);
```

Comment:
Please define these constants inside the `App` class.

Reply:
For sure? 

I saw it at 
https://github.com/friendica/friendica/blob/8a7c9d6666336eddc877e57ec134e585b4edcf84/src/Network/FKOAuthDataStore.php#L19
and thought I do it the same way ;-)

Reply:
😬😬😬

This class is part of the "legacy code I don't want to touch because I'm not sure it would keep working if it ever did".

Reply:
ah, oh... I see ;-)

So I will add it to the class and make some useful methods around it